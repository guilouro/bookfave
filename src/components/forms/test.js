import React from 'react'
import { render } from 'react-testing-library'
import Forms from '.'

describe('<Forms />', () => {
  it('Should render correctly', () => {
    const { container } = render(<Forms />)
    expect(container).toMatchSnapshot()
  })
})
