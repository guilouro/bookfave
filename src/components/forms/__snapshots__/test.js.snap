// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Forms /> Should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  width: 70px;
  height: 35px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-right: 10px;
}

.c2 {
  outline: none;
  border: none;
  background: transparent;
  margin: 0;
  padding: 0;
  cursor: pointer;
  position: absolute;
  color: #b0b9bf;
  -webkit-transition: 0.5s all;
  transition: 0.5s all;
  -webkit-transform: translateX(-100%);
  -ms-transform: translateX(-100%);
  transform: translateX(-100%);
}

.c3 {
  outline: none;
  border: none;
  background: transparent;
  margin: 0;
  padding: 0;
  cursor: pointer;
  position: absolute;
  color: #1e68c1;
  -webkit-transition: 0.5s all;
  transition: 0.5s all;
  -webkit-transform: translateX(80%);
  -ms-transform: translateX(80%);
  transform: translateX(80%);
}

.c4 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c4 button {
  background-color: #1e68c1;
  color: #fff;
  border: none;
  padding: 10px;
  border-radius: 5px;
}

.c4 button:disabled {
  background-color: #b0b9bf;
}

.c5 {
  border: 1px solid #e7e8eb;
  border-radius: 4px;
  padding: 8px 10px;
  margin-bottom: 10px;
}

.c5::-webkit-input-placeholder {
  color: #b0b9bf;
}

.c5::-moz-placeholder {
  color: #b0b9bf;
}

.c5:-ms-input-placeholder {
  color: #b0b9bf;
}

.c5::placeholder {
  color: #b0b9bf;
}

@media (min-width:900px) {
  .c0 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

@media (min-width:600px) {
  .c4 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

@media (min-width:900px) {
  .c4 button {
    padding-top: 0;
    padding-bottom: 0;
  }
}

@media (min-width:600px) {
  .c5 {
    -webkit-flex: 1;
    -ms-flex: 1;
    flex: 1;
    margin: 0;
  }

  .c5:not(:last-child) {
    margin-right: 15px;
  }
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <button
      class="c2"
    >
      <svg
        height="20px"
        viewBox="0 0 26 26"
        width="20px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.583 20.083c-4.687 0-8.5-3.812-8.5-8.5 0-4.687 3.813-8.5 8.5-8.5 4.688 0 8.5 3.813 8.5 8.5 0 4.688-3.812 8.5-8.5 8.5m13.752 3.249l-4.81-4.811a11.265 11.265 0 0 0 2.391-6.938C22.916 5.334 17.832.25 11.583.25 5.334.25.25 5.334.25 11.583c0 6.25 5.084 11.334 11.333 11.334 2.615 0 5.02-.9 6.937-2.392l4.811 4.81a1.413 1.413 0 0 0 2.004 0 1.415 1.415 0 0 0 0-2.003"
          fill="currentColor"
        />
      </svg>
    </button>
    <button
      class="c3"
    >
      <svg
        height="20px"
        viewBox="0 0 26 26"
        width="20px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M18.121 11.72h-3.84V7.879a1.28 1.28 0 0 0-2.561 0v3.84H7.879a1.28 1.28 0 0 0 0 2.561h3.84v3.841a1.28 1.28 0 0 0 2.561 0v-3.84h3.841a1.28 1.28 0 0 0 0-2.561zm5.121 7.682a3.845 3.845 0 0 1-3.84 3.84H6.598a3.845 3.845 0 0 1-3.84-3.84V6.598a3.845 3.845 0 0 1 3.84-3.84h12.804a3.845 3.845 0 0 1 3.84 3.84v12.804zM19.402.197H6.598A6.409 6.409 0 0 0 .197 6.598v12.804a6.409 6.409 0 0 0 6.401 6.401h12.804a6.409 6.409 0 0 0 6.401-6.401V6.598A6.409 6.409 0 0 0 19.402.197z"
          fill="currentColor"
        />
      </svg>
    </button>
  </div>
  <form
    novalidate=""
  >
    <div
      class="c4"
    >
      <input
        class="c5"
        name="title"
        placeholder="Title"
        required=""
        value=""
      />
      <input
        class="c5"
        name="link"
        placeholder="Link"
        required=""
        value=""
      />
      <input
        class="c5"
        name="tags"
        placeholder="Tags"
        value=""
      />
      <button
        type="submit"
      >
        Add
      </button>
    </div>
  </form>
</div>
`;
